### PR TITLE
Refresh user dashboard feed section after login

### PR DESCRIPTION
**Pull request by mzparacha**
## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/3946

## Description of Changes
Refreshes the user dashboard feed sections after login

## Test Plan
1. Go to the /dashboard page
2. Logout then login
3. Verify that the feed sections get updated

## Deployment Plan
When a user logouts on `/dashboard/for-you`, it redirects to `/dashboard/global` which seems like a bad UX. I asked a question for this [here](https://github.com/hicommonwealth/commonwealth/issues/3946#issuecomment-1618732475), awaiting how to move forward here. @CowMuon @jnaviask 

**Update:**
Got the reply [here](https://github.com/hicommonwealth/commonwealth/issues/3946#issuecomment-1629612670) which suggested the same implementation as in the pr. PR is ready for review

## Other Considerations
N/A

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/pages/user_dashboard/index.tsx [hunk 1]
```diff
@@ -23,6 +24,7 @@ type UserDashboardProps = {
 
 const UserDashboard = (props: UserDashboardProps) => {
   const { type } = props;
+  const {isLoggedIn} = useUserLoggedIn();
```
Comment (jnaviask) [inline packages/commonwealth/client/scripts/views/pages/user_dashboard/index.tsx:27]: Not linted / formatted properly? Should have spaces around the object destructuring.

File: packages/commonwealth/client/scripts/views/pages/user_dashboard/index.tsx [hunk 2]
```diff
@@ -64,7 +66,7 @@ const UserDashboard = (props: UserDashboardProps) => {
   }, [activePage, subpage]);
 
   return (
-    <div ref={setScrollElement} className="UserDashboard">
+    <div ref={setScrollElement} className="UserDashboard" key={isLoggedIn + ''}>
```
Comment (jnaviask) [inline packages/commonwealth/client/scripts/views/pages/user_dashboard/index.tsx:69]: We should use `${isLoggedIn}` rather than adding to ''.

**Review by jnaviask — CHANGES_REQUESTED** [review]
Logic seems fine but some small issues.

**Comment by jnaviask** [comment]
cc @mzparacha to respond to comments / fix

**Comment by mzparacha** [comment]
Fixed cc: @jnaviask

**Review by jnaviask — APPROVED** [review]

**Pull request merged**